### PR TITLE
Rewrite article that describes how to implement federated authentication between SPS and AD FS

### PR DESCRIPTION
**Pull request by Yvand**
Current version of [this article](https://docs.microsoft.com/en-us/sharepoint/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server) has many issues:
- It does not describe the configuration needed/expected on AD FS
- It contains a lot of unnecessary text
- The whole SharePoint configuration is completely wrong and confusing. I'll just highlights a few points:
	- It adds multiple certificates SPTrustedRootAuthority, some of which are totally useless
	- The script that creates the SPTrustedIdentityTokenIssuer is very confusing and has a lot of readability and practical issues
	- It misses the configuration of the web application, the IIS site and the creation of the site collection, while it contains unnecessary/irrelevant chapters (on tracing and audiences)

I preferred to rewrite it entirely to create a clear step by step to configure federated authentication end to end, including AD FS. I used an approach very similar to [this article](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/application-proxy-integrate-with-sharepoint-server) that I rewrote a few years ago.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 1]
```diff
@@ -14,260 +14,229 @@ ms.collection:
...
-## Concepts and terminology ##
-To understand the concepts and terminology that are used in SAML-based authentication, see [Authentication Overview](https://docs.microsoft.com/sharepoint/security-for-sharepoint-server/authentication-overview).
+In federated authentication, SharePoint processes SAML tokens issued by a trusted, external Security Token Service (STS). A user who attempts to log on is redirected to that STS, which authenticates the user and generates a SAML token upon successful authentication. Then SharePoint processes this token, and uses it to create its own and authorize the user to access the site.
 
+## Prerequisites
 
-## SharePoint Server with Active Directory Federation Services 2.0 ##
+To perform the configuration, you need the following resources:
+- A SharePoint 2013 farm or newer.
+- An AD FS farm version 2 or newer, already created, with the public key of the ADFS signing certificate exported in a .cer file
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md:35]:
```suggestion
- An AD FS farm version 2 or newer, already created, with the public key of the AD FS signing certificate exported in a .cer file.
```

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 2]
```diff
@@ -14,260 +14,229 @@ ms.collection:
...
-
-4.	On the **Start** menu, click **All Programs** > **Microsoft SharePoint Products** > **SharePoint  Management Shell**.
-5.	Change the command prompt to the directory to which you saved the file.
-6.	At the Windows PowerShell command prompt, type the following command:  ./Add-ADFSCerts.ps1.
+> [!IMPORTANT]
+> Do NOT use the option -UseDefaultConfiguration with cmdlet New-SPTrustedIdentityTokenIssuer. This option causes unexpected side effects due to the way it sets the identity of the users internally.
 
-## Create the trusted provider by using a Windows PowerShell script ##
+Then, the relevant certificate must be added to the SharePoint root authority certificate store. There are 2 possible options:
+- If the ADFS signing certificate is issued by a certificate authority (best practice for security reasons)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md:112]:
```suggestion
- If the AD FS signing certificate is issued by a certificate authority (best practice for security reasons)
```

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 3]
```diff
@@ -14,260 +14,229 @@ ms.collection:
...
+    1. Start the SharePoint Management Shell and run the following script:
 
-1.	Check that you meet the following minimum requirements:
-- See [Add-SPShellAdmin](https://docs.microsoft.com/powershell/module/sharepoint-server/add-spshelladmin?view=sharepoint-ps).
-- Read [about_Execution_Policies](https://go.microsoft.com/fwlink/p/?LinkId=193050).
+        ```powershell
+        # This script extends an existing web application to set AD FS authentication on a new zone
+        # URL of the default zone of the web application
+        $webAppDefaultZoneUrl = "http://spsites/"
+        # URL of the SharePoint site federated with ADFS
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md:172]:
```suggestion
        # URL of the SharePoint site federated with AD FS
```

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 4]
```diff
@@ -14,260 +14,229 @@ ms.collection:
...
+1. Open the Internet Information Services Manager console.
+1. Expand the server in the tree view, expand **Sites**, select the **SharePoint - ADFS on contoso.local** site, and select **Bindings**.
+1. Select **https binding** and then select **Edit**.
+1. In the TLS/SSL certificate field, choose **spsites.contoso.local** certificate and then select **OK**.
 
-## Trusted identity providers and user profile synchronization ##
+## Create the site collection
 
-This section describes the steps that let user profile synchronization use claims-based authentication with a trusted identity provider.
+In this step, you create a team site collection with 2 administrators: One as a Windows administrator and one as a federated (AD FS) administrator.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md:215]:
```suggestion
In this step, you create a team site collection with two administrators: One as a Windows administrator and one as a federated (AD FS) administrator.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor edits.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Yvand** [comment]
@scanum what are the next steps for this PR?

**Comment by Yvand** [comment]
@scanum can you follow up this PR ?

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@SerdarSoysal

**Comment by Yvand** [comment]
@SerdarSoysal any update on this?

**Pull request merged**